ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -362,12 +366,18 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
             tap(() => ++initiatedVerifyAddressCalls),
             switchMap(inputAddress =>
                 this.addressService.verifyAddress(inputAddress).pipe(
+                    tap(_ => {
+                        this.block.hasValidAddress = false;
```

Comment:
do we need this tap here? in the next tap operator we set hasValidAddress as true and it's guaranteed

Reply:
Good point.
Moved it before `this.addressService.verifyAddress`, that was the intention.